src/cli/README.md

```diff
@@ -1049,6 +1050,52 @@ Set time sync parameters
 Done
 ```
 
+### debug
+
+Executes a series of CLI commands to gather information about the device and thread network. This is intended for debugging.
+
+The output will display each executed CLI command preceded by "\$", followed by the corresponding command's generated output.
+
+The generated output encompasses the following information:
+
+- Version
+- Current state
+- RLOC16, Extended MAC address
+- Unicast and multicast IPv6 address list
+- Channel
+- PAN ID and Extended PAN ID
+- Network Data
+- Partition ID
+- Leader Data
+
+If device is operating as FTD:
+
+- Child and Neighbor table
+- Router Table and Next Hop Info
```

Comment:
recommend consistency with whether or not all words for each item are in upper or lower case